datalad/core/local/create.py

```diff
@@ -206,6 +209,9 @@ def __call__(
         # we know that we need to create a dataset at `path`
         assert(path is not None)
 
+        # assure cfg_proc is a list (relevant if used via Python API)
```

Comment:
Oh no, the misnamed function has infected your comment :]  You're not trying to ease this list's mind or give it confidence about something.  You're forcibly casting it to a list.  Ensure!